org.eclipse.bluesky/language-clients/css/language-configuration.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
On all those, files, I believe it is important to place a comment or other kind of annotation to mention that those are taken from VSCode.

Reply:
It's a JSON file, how can we write comments?

Reply:
Forget my comment, GSON is able to parse comments even if comment is not in the JSON specification. I have added license, etc